libs/langchain-google-common/src/chat_models.ts

```diff
@@ -398,8 +351,10 @@ export abstract class ChatGoogleBase<AuthOptions>
                 usage_metadata: usageMetadata,
               }),
             });
-      yield chunk;
-      await runManager?.handleLLMNewToken(chunk.text);
+      if (chunk) {
+        yield chunk;
+        await runManager?.handleLLMNewToken(chunk.text);
```

Comment:
Unrelated nit: good to pass the full generation chunk here for other things

For example:

https://github.com/langchain-ai/langchainjs/blob/main/libs/langchain-openai/src/chat_models.ts#L1427

Reply:
I think I'm missing something about the nit. What about the "full generation chunk" isn't being passed here?

Reply:
Ah! Got it! Fixed.